### PR TITLE
Emily's Homework (Fingers Crossed)

### PR DESCRIPTION
**Pull request by EmilyForden**

### CODE REVIEW

**Comment by EmilyForden** [comment]
I decided to take a break and a walk to clear my head. Now I have tried to fork, clone, edit, pull, and push. I hope this works!

**Comment by EmilyForden** [comment]
[https://raw.githubusercontent.com/EmilyForden/hw01/34912b1f46a5cfa87df66d194bf4f9cfa028e716/Emily%20Forden%20HW1.R](url)

**Comment by bensoltoff** [comment]
Sounds good! Just a head's up
1. If you want your documents to render on Github, save them as a `.md` file (for "markdown"). Anything with a `.r` extension is treated as a [script](http://r4ds.had.co.nz/workflow-scripts.html) for R.
2. Once you submit a pull request, don't close it. Otherwise I won't see it to grade. You can post a comment on the pull request and still keep the PR open.

**Comment by EmilyForden** [comment]
WOW! I think I did it! I can see the picture of my super cute cat, which means that the HTML is working! Here is everything I did to complete this homework assignment:
1. Downloaded the master (not the fork) README.md, which meant that I couldn't upload the changes that I made. 
1a. Got frustrated.
2. Realized the mistake and forked the homework and then cloned it to my computer. 
3. Edited my profile using markdown. 
3a. Got frustrated.
4. Uploaded the file in the wrong format and learned now to close a pull request because it blocks others from seeing my work. 
5. Figured out how to save something as an .md file and then committed that. 
6. Saw it work! There is my picture of Otho!
7. Saw a typo, so I re-commited. 

Reflections: 
1. GitHub likes things a very certain way and you have to remember to obey it at all times. I made a lot of mistakes, but I think I was able to fix them and clean up my digital trail. For example, I learned how to delete a repo. I also learned how to comment on a repo. 
2. Markdown is super easy to use and I really like it. I find the commands to be pretty intuitive and powerful. I still have a few questions about it though:
A. I couldn't get a reference style link to work. I think a reference style is something like the source of my photo. In this example, I took the photo of Otho but if I wanted to credit someone else, I don't know that I know how. 
B. How can I change the color of my words, say, so that each was a different color? Is there a more advanced cheatsheet?

File: Emily Forden HW1.R [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Hello! My name is *Emily Forden* and I am a 
+**5th Year** in the 
+[History PdD program](https://history.uchicago.edu/) 
+where I study ancient Roman economics.
+
+_Here are some interesting things about me!_
+1. I got married in June!
+2. I have a cat named Otho, who was a Roman Emperor. 
+Here is a photo of him:
+![alt text](http://i.imgur.com/maQQvXy.jpg)
```
Comment (bensoltoff) [inline Emily Forden HW1.R:11]: Yay pictures! Also a cute cat. Here's mine (Ariel)

Ariel the cat

File: Emily Forden HW1.R [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+I'm working on becoming a dive master.
+
+##Thoughts and Reflections
+Although I found it easy to edit a clone on my computer, 
+I had a really hard time getting
+the document to appear in GitHub. At times, I could 
+be heard muttering:
+  
+>I'm going to smash my computer
+>like the Hulk
```
Comment (bensoltoff) [inline Emily Forden HW1.R:22]: Could condense into one line and get the same output

`> I'm going to smash my computer like the hulk`

> I'm going to smash my computer like the hulk

File: Emily Forden HW1.R [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#Introduction
```
Comment (bensoltoff) [inline Emily Forden HW1.R:1]: Probably best to delete this file once you created the contents in `README.md` - it's extraneous, and not actually an `.R` script

**Comment by bensoltoff** [comment]
## Evaluation HW-01 for Emily Forden

| Topic | Excellent | Satisfactory | Needs Work |
| --- | --- | --- | --- |
| **Coding style** | x |  |  |
| **Coding strategy** |  | x |  |
| **Presentation: graphs** | x |  |  |
| **Presentation: tables** |  |  |  |
| **Achievement, creativity** | x |  |  |
| **Ease of access** |  | x |  |

Remarks:
- Lots of content in your readme, and you used markdown syntax. Great!
- No tables (as expected)
- Lots of commits - that's good to keep track of changes and figure out what worked and what didn't
- Next time, write your reflection as a comment on the pull request, not in the main doc

**Pull request closed without merging**